components/BottomNavigation/src/private/MDCBottomNavigationItemBadge.m

```diff
@@ -11,12 +11,16 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+#import <CoreGraphics/CoreGraphics.h>
 
 #import "MDCBottomNavigationItemBadge.h"
+#import "MaterialMath.h"
 
-static const CGFloat kMDCBottomNavigationItemBadgeFontSize = 10;
-static const CGFloat kMDCBottomNavigationItemBadgeXPadding = 8;
-static const CGFloat kMDCBottomNavigationItemBadgeYPadding = 2;
+static const CGFloat kBadgeFontSize = 10;
+// These padding values get pretty close to the material.io guidelines article.
+static const CGFloat kBadgeYPadding = 2;
+// For an empty badge, ensure that the size is close to the guidelines article.
+static const CGFloat kMinDiameter = 9;
```

Comment:
_Question:_ Where is this value coming from?

Reply:
Just eyeballed it to get as close to the mocks as possible.

Reply:
Sounds good to me